quickstarts/releaseiq/dashboard/riq-dashboard.json

```diff
@@ -0,0 +1,157 @@
+{
+  "name": "RIQ Dashboard",
+  "description": "",
+  "permissions": "PUBLIC_READ_WRITE",
```

Comment:
```suggestion
```

Reply:
You don't want to include this in the JSON. In the future you can run our sanitize script to remove things such as this. https://github.com/newrelic/newrelic-quickstarts#-dashboards